src/main/drivers/rx/rx_sx1280.h

```diff
@@ -0,0 +1,282 @@
+    SX1280_LORA_PACKET_EXPLICIT = SX1280_LORA_PACKET_VARIABLE_LENGTH,
+    SX1280_LORA_PACKET_IMPLICIT = SX1280_LORA_PACKET_FIXED_LENGTH,
+} sx1280_lora_packet_lengths_modes_e;
+
+typedef enum {
+    SX1280_LORA_CRC_ON = 0x20,  // CRC activated
+    SX1280_LORA_CRC_OFF = 0x00, // CRC not used
+} sx1280_lora_crc_modes_e;
+
+typedef enum {
+    SX1280_RADIO_GET_STATUS = 0xC0,
+    SX1280_RADIO_WRITE_REGISTER = 0x18,
+    SX1280_RADIO_READ_REGISTER = 0x19,
+    SX1280_RADIO_WRITE_BUFFER = 0x1A,
+    SX1280_RADIO_READ_BUFFER = 0x1B,
+    SX1280_RADIO_SET_SLEEP = 0x84,
+    SX1280_RADIO_SET_STANDBY = 0x80,
+    SX1280_RADIO_SET_FS = 0xC1,
+    SX1280_RADIO_SET_TX = 0x83,
+    SX1280_RADIO_SET_RX = 0x82,
+    SX1280_RADIO_SET_RXDUTYCYCLE = 0x94,
+    SX1280_RADIO_SET_CAD = 0xC5,
+    SX1280_RADIO_SET_TXCONTINUOUSWAVE = 0xD1,
+    SX1280_RADIO_SET_TXCONTINUOUSPREAMBLE = 0xD2,
+    SX1280_RADIO_SET_PACKETTYPE = 0x8A,
+    SX1280_RADIO_GET_PACKETTYPE = 0x03,
+    SX1280_RADIO_SET_RFFREQUENCY = 0x86,
+    SX1280_RADIO_SET_TXPARAMS = 0x8E,
+    SX1280_RADIO_SET_CADPARAMS = 0x88,
+    SX1280_RADIO_SET_BUFFERBASEADDRESS = 0x8F,
+    SX1280_RADIO_SET_MODULATIONPARAMS = 0x8B,
+    SX1280_RADIO_SET_PACKETPARAMS = 0x8C,
+    SX1280_RADIO_GET_RXBUFFERSTATUS = 0x17,
+    SX1280_RADIO_GET_PACKETSTATUS = 0x1D,
+    SX1280_RADIO_GET_RSSIINST = 0x1F,
+    SX1280_RADIO_SET_DIOIRQPARAMS = 0x8D,
+    SX1280_RADIO_GET_IRQSTATUS = 0x15,
+    SX1280_RADIO_CLR_IRQSTATUS = 0x97,
+    SX1280_RADIO_CALIBRATE = 0x89,
+    SX1280_RADIO_SET_REGULATORMODE = 0x96,
+    SX1280_RADIO_SET_SAVECONTEXT = 0xD5,
+    SX1280_RADIO_SET_AUTOTX = 0x98,
+    SX1280_RADIO_SET_AUTOFS = 0x9E,
+    SX1280_RADIO_SET_LONGPREAMBLE = 0x9B,
+    SX1280_RADIO_SET_UARTSPEED = 0x9D,
+    SX1280_RADIO_SET_RANGING_ROLE = 0xA3,
+} sx1280_commands_e;
+
+typedef enum {
+    SX1280_IRQ_RADIO_NONE = 0x0000,
+    SX1280_IRQ_TX_DONE = 0x0001,
+    SX1280_IRQ_RX_DONE = 0x0002,
+    SX1280_IRQ_SYNCWORD_VALID = 0x0004,
+    SX1280_IRQ_SYNCWORD_ERROR = 0x0008,
+    SX1280_IRQ_HEADER_VALID = 0x0010,
+    SX1280_IRQ_HEADER_ERROR = 0x0020,
+    SX1280_IRQ_CRC_ERROR = 0x0040,
+    SX1280_IRQ_RANGING_SLAVE_RESPONSE_DONE = 0x0080,
+    SX1280_IRQ_RANGING_SLAVE_REQUEST_DISCARDED = 0x0100,
+    SX1280_IRQ_RANGING_MASTER_RESULT_VALID = 0x0200,
+    SX1280_IRQ_RANGING_MASTER_TIMEOUT = 0x0400,
+    SX1280_IRQ_RANGING_SLAVE_REQUEST_VALID = 0x0800,
+    SX1280_IRQ_CAD_DONE = 0x1000,
+    SX1280_IRQ_CAD_DETECTED = 0x2000,
+    SX1280_IRQ_RX_TX_TIMEOUT = 0x4000,
+    SX1280_IRQ_PREAMBLE_DETECTED = 0x8000,
+    SX1280_IRQ_RADIO_ALL = 0xFFFF,
+} sx1280_irq_masks_e;
+
+typedef enum {
+    SX1280_RADIO_DIO1 = 0x02,
+    SX1280_RADIO_DIO2 = 0x04,
+    SX1280_RADIO_DIO3 = 0x08,
+} sx1280_dios_e;
+
+typedef enum {
+    SX1280_RADIO_TICK_SIZE_0015_US = 0x00,
+    SX1280_RADIO_TICK_SIZE_0062_US = 0x01,
+    SX1280_RADIO_TICK_SIZE_1000_US = 0x02,
+    SX1280_RADIO_TICK_SIZE_4000_US = 0x03,
+} sx1280_tick_sizes_e;
```

Comment:
Still a lot of snake_case here and in rx_sx1280.c

Reply:
Sorry I was looking at one file at a time, too many changes. Fixed.